src/main/java/com/denizenscript/clientizen/scripts/commands/GuiCommand.java

```diff
@@ -16,29 +18,33 @@ public class GuiCommand extends AbstractCommand implements Holdable {
 
     // <--[command]
     // @Name gui
-    // @Syntax gui [<script>]
+    // @Syntax gui [open <script>]/[close]
     // @Required 1
-    // @Maximum 1
-    // @Short Opens a GUI script into a GUI.
+    // @Maximum 2
+    // @Short Opens a GUI script into a GUI or closes a currently open one.
```

Comment:
maybe just "Opens or closes a GUI script"

Reply:
Worded it that way because it can technically close any open GUI, not just ones from GUI scripts